terraform/environments/core-network-services/firewall-rules/preproduction_rules.json

```diff
@@ -454,11 +454,123 @@
+    "protocol": "UDP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_123_NTP_UDP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "123",
+    "protocol": "UDP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_135_MS_RPC_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "135",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_139_NetBios_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "139",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_389_LDAP_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "389",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_389_LDAP_UDP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "389",
+    "protocol": "UDP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_445_SMB_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "445",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_464_Kerberos_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "464",
+    "protocol": "UDP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_464_Kerberos_UDP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "464",
+    "protocol": "UDP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_636_Kerberos_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "636",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_3268_LDAP_glocal_catalogue_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "3268",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_3269_LDAPS_TCP": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction-general-private-subnets}",
+    "destination_ip": "${mojo-end-user-devices}",
+    "destination_port": "3269",
+    "protocol": "TCP"
+  },
+  "dom1_dcs_to_planetfm_preproduction_9389_LDAPS_TCP": {
```

Comment:
this is ADWS (active directoy web services)

Reply:
Updated to correct name